### PR TITLE
review by UA of index.md

### PR DESCRIPTION
**Pull request by jhornik**
UA review up until "Creating an MTA Archive According to the MTA Deployment Descriptor".

Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
 
-If the MTA Build Tool encounters a module type in the `mta.yaml` that is not listed in the configuration file, its definition is copied to `mtad.yaml` as is and the `zip` builder is applied to the module (i.e. archived module sources will be packaged as the build result).
+For more details about how the design-time module types in the `mta.yaml` file are converted into deployment time module types in the `mtad.yaml` file according to the target platform, see [Module types conversion](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/platform_cfg.yaml). 
 
-All resource definitions are passed to the `mtad.yaml` file as is without mapping and validations (e.g., if the resource is supported in the target platform or specified parameters match the type).
+If the MTA Build Tool encounters a module type in the `mta.yaml` file that is not listed in the configuration file, its definition is copied to the `mtad.yaml` file as is and the `.zip` builder is applied to the module (in other words, the archived module sources are packaged as the build result).
```
Comment (katunin-ng) [inline docs/docs/index.md]: "zip" is the actual name of the builder. So it should not be replaced with ".zip"

Comment (jhornik) [reply docs/docs/index.md]: Ok, thanks for the info. Will you make the change or should I?

Comment (katunin-ng) [reply docs/docs/index.md]: Please change it. Thank you

Comment (jhornik) [reply docs/docs/index.md]: I cannot see a way to make that change now. I suggest that @ShimiT should merge the changes and then I will do an extra review.

Comment (katunin-ng) [reply docs/docs/index.md]: No problem. I will change it.

Comment (jhornik) [reply docs/docs/index.md]: Thanks

**Pull request merged**